<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cap_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>물고기 사진 촬영</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload_btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>기존 사진 업로드</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
